src/main/java/org/folio/service/orders/flows/update/unopen/UnOpenCompositeOrderManager.java

```diff
@@ -169,10 +169,6 @@ private Future<Void> processInventoryOnlyWithHolding(CompositePoLine compPOL, Re
   }
 
   private Future<Void> processInventoryOnlyWithItems(CompositePoLine compPOL, RequestContext requestContext) {
-    if (!PoLineCommonUtil.isReceiptNotRequired(compPOL.getReceiptStatus())) {
```

Comment:
**Question**
Could you please explain removing this check?

Reply:
it is conflicting check with the check that we have in deleteExpectedPieces, and after fixing to make it similar it is not makes sense to doube it with deleteExpectedPieces.
This check was added previosly in scope of some refactoring and it was wrong
Also please see https://github.com/folio-org/mod-orders/blob/master/src/main/java/org/folio/service/pieces/flows/strategies/ProcessInventoryStrategy.java#L56 - this check is used to create/update pieces, so for deleting them we also should have the similar. I doubted between OR or AND but remained AND as it was previously